Xamarin.PropertyEditing.Windows/Themes/Resources.xaml

```diff
@@ -296,6 +297,57 @@
 		</Setter>
 	</Style>
 
+	<system:Double x:Key="MenuOffset">3</system:Double>
+
+	<Rectangle x:Shared="False" x:Key="LiteralMarker" Stroke="{DynamicResource PropertyButtonBorderBrush}" Fill="{DynamicResource LiteralMarkerBrush}" Height="12" Width="12" StrokeThickness="1" />
+
+	<Style TargetType="local:PropertyButton">
+		<Setter Property="Background" Value="Transparent" />
+		<Setter Property="BorderThickness" Value="0" />
+		<Setter Property="Height" Value="17" />
+		<Setter Property="Width" Value="11" />
+		<Setter Property="SnapsToDevicePixels" Value="True" />
+		<Setter Property="MenuTemplate">
+			<Setter.Value>
+				<DataTemplate>
+					<local:HeaderedContextMenu Header="{Binding Property.Name,Mode=OneTime}" StaysOpen="True" Placement="Left" HorizontalOffset="{StaticResource MenuOffset}" VerticalOffset="{StaticResource MenuOffset}">
+						<!--<MenuItem x:Name="CustomExpressionItem" Header="{x:Static prop:Resources.CustomExpressionEllipsis}" Icon="{StaticResource LiteralMarker}" />
```

Comment:
Any particular reason to keep commented out code around?

Reply:
We'll be adding them back as we have support for them.